packages/request-external-access/package.json

```
@@ -0,0 +1,39 @@
+{
+	"name": "@automattic/request-external-access",
+	"version": "1.0.0",
+	"description": "Utility for requesting authorization of sharing services.",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"sideEffects": false,
+	"keywords": [
+		"wordpress",
+		"access",
+		"popup"
+	],
+	"author": "Automattic Inc.",
+	"homepage": "https://github.com/Automattic/wp-calypso/tree/master/packages/request-external-access",
+	"license": "GPL-2.0-or-later",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/request-external-access"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"files": [
+		"dist",
+		"src"
+	],
+	"scripts": {
+		"clean": "npx rimraf dist",
+		"prepublish": "yarn run clean",
+		"prepare": "transpile"
+	},
+	"dependencies": {
+		"@automattic/popup-monitor": "1.0.0"
```

We are trying to not pin dependencies in `package.json` unless there is a very good reason
```suggestion
		"@automattic/popup-monitor": "^1.0.0"
```

---

How come this new dependency didn't change `yarn.lock`?